packages/extension-ui/src/Popup/Accounts/AddAccountMenu.tsx

```diff
@@ -17,6 +20,13 @@ interface Props extends ThemeProps {
 
 function AddAccountMenu({ className }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
+  const onAction = useContext(ActionContext);
+
+  const _goToAccountCreate = useCallback(() => onAction('/account/create'), [onAction]);
+
+  const _goHome = useCallback(() => onAction('/'), [onAction]);
+
+  const _goToImportSeed = useCallback(() => onAction('/account/import-seed'), [onAction]);
```

Comment:
Wouldn't it work to 
```
const goTo = useCallback((path) => () => onAction(path), [onAction]);
```
and then in template
```
onClick={goTo('/account/create')}
```
etc for the rest? Would make code a bit DRYier if linter would allow it.